definitions/infra-kubernetes_pod/definition.yml

```diff
@@ -31,5 +31,6 @@ synthesis:
         multiValue: false
     newrelic.agentVersion:
         multiValue: false
+    host.instanceType:
     aws.region:
-    aws.ec2.instanceType:
+      multiValue: false
```

Comment:
is the indentation right here? (I see it +2 chars deeper in L33)

Reply:
Fixing this. The standard seems to be 2 spaces, right? Just confirming.